test/execute_build_test.go

```diff
@@ -131,6 +116,11 @@ func testCreateImage(t *testing.T, when spec.G, it spec.S) {
 					Name: dockerSecret,
 				},
 			},
+			ImagePullSecrets: []corev1.LocalObjectReference{
```

Comment:
Was this driven by testing this in an environment that needed pull secrets?

Reply:
ya, I wanted it to be easy for anyone to run the tests on their own deployment